lib/granite/base.rb

```diff
@@ -0,0 +1,37 @@
+require 'active_data/model'
+require 'active_data/model/primary'
+require 'active_data/model/lifecycle'
+require 'active_data/model/associations'
+
+require 'granite/translations'
+require 'granite/represents'
+
+module Granite
+  module Base
+    extend ActiveSupport::Concern
+
+    include ActiveSupport::Callbacks
+    include ActiveData::Model
+    include ActiveData::Model::Representation
+    include ActiveData::Model::Dirty
+    include ActiveData::Model::Associations
+    include ActiveData::Model::Primary
+    include ActiveModel::Validations::Callbacks
+
+    include Granite::Translations
+    include Granite::Represents
+
+    included do
+      def self.i18n_scope
+        :granite_action
```

Comment:
I guess this shouldn't be a `granite_action` since it's not an action. Can we declare it in `Granite::Action`?